garden_ai/garden.py

```diff
@@ -79,3 +81,65 @@ def _authenticate(self):
             on_refresh=self.auth_key_store.on_refresh,
         )
         return authorizer
+
+    def create_garden(self, **kwargs):
+        """Initialize and return a new Garden object
```

Comment:
some more clarity in the docstring on what a Garden object is would be helpful

Reply:
That is an excellent question that I think is also tied to the above comment about whether or not `create_garden()` should just be a regular constructor for a class. 

In the past, we have used names like `FoundryMetadata` or `FoundryBaseModel` to describe Pydantic classes describing the metadata, where then `Foundry` might be the class that describes the actual Foundry object. 

I am not as familiar with Pydantic, so my suggestion would be to read into what the best practices are for how to conceptually organize this to be in-line with OOP principles. 

A major question here is we know that the Pydantic class describes the state (ie attributes) of a Garden, but what about the function? Does that go in another `Garden` class, or just in the `GardenClient`? 

If there are no clear answers, I'd lean towards the Pydantic class being called `GardenMetadata` or something similar, and then having the actual object (which contains `GardenMetadata` as an attribute) be `Garden` or `GardenClient`, depending on where we want function to live.

Reply:
Context so I can resolve this in good conscience: the current `Garden` object is on track to be greater than the sum of its metadata, with more functionality than would be provided by a more vanilla pydantic model. This feels ok to me, but I do think you're right here to question whether this is good OOP, & my feelings won't be hurt if we later decide that the interface would be better if garden metadata and ~gardening functionality were owned by different classes